.circleci/config.yml

```diff
@@ -175,6 +231,12 @@ workflows:
       - test-windows
       - test-browser
       - lint
+      - build-docker:
+         filters:
+            branches:
+              only:
+                - develop
+                - chore/can-we-build-docker-in-circle
```

Comment:
```suggestion
```

Reply:
We currently build it for the `develop` branch though @P0lip

Reply:
Hmm, do we need to, tho? I'd say we should only run it when a given commit is tagged appropriately

Reply:
at least line 239 should be removed :laughing:

Reply:
I mean, honestly though line no 239 is totally harmless and there's no reason to remove it. :D

Buuuut yeah honestly I'm OK with no branch builds either. The only question then is do we always tag every new release with `latest` instead of doing a separate branch build on `master` or `main` for that.

Reply:
The more I think about it, the more inclined I am to simply remove the `latest` tag. :/ No one in their right mind would use it anyway since it jumps across major versions.